package.json

```diff
@@ -19,7 +19,7 @@
     "test": "kcd-scripts test",
     "validate": "kcd-scripts validate",
     "commit": "git-cz",
-    "start": "./dist/cli.js",
+    "start": "node ./dist/cli.js",
```

Comment:
`node` isn't needed here since the `cli.js` is an executable script and no one I know had issues running it since I last checked.

Reply:
I had issue on my environment, buy I'm using Cygwin which is not officially supported by npm,
so I'm totally fine to remove this